Makefile

```diff
@@ -451,6 +453,47 @@ st-checks:
 	# running on the host.
 	iptables-save | grep -q 'calico-st-allow-etcd' || iptables $(IPT_ALLOW_ETCD)
 
+## Get the kubeadm-dind-cluster script
+tests/k8st/dind-cluster.sh:
+	wget -O tests/k8st/dind-cluster.sh https://raw.githubusercontent.com/neiljerram/kubeadm-dind-cluster/master/dind-cluster.sh
+	chmod +x tests/k8st/dind-cluster.sh
+
+.PHONY: k8s-test
+## Run the k8s tests
+k8s-test: $(NODE_CONTAINER_CREATED) calico_test.created tests/k8st/dind-cluster.sh k8s-stop k8s-start
+	docker run \
+	    -v $(CURDIR):/code \
+	    -v /var/run/docker.sock:/var/run/docker.sock \
+	    -v /home/$(USER)/.kube/config:/root/.kube/config \
+	    -v /home/$(USER)/.kubeadm-dind-cluster:/root/.kubeadm-dind-cluster \
+	    --privileged \
+	    --net host \
+        $(TEST_CONTAINER_NAME) \
+	    sh -c 'cp /root/.kubeadm-dind-cluster/kubectl /bin/kubectl && ls -ltr /bin/kubectl && which kubectl && cd /code/tests/k8st && nosetests -v --with-xunit --xunit-file="/code/report/k8s-tests.xml" --with-timer'
+	$(MAKE) k8s-stop
+
+.PHONY: k8s-start
+## Start k8s cluster
+k8s-start: tests/k8st/dind-cluster.sh calico-node.tar
+	CNI_PLUGIN=custom POD_CIDR=192.168.0.0/16 SKIP_SNAPSHOT=y tests/k8st/dind-cluster.sh up
+	kubectl apply -f  https://docs.projectcalico.org/master/getting-started/kubernetes/installation/hosted/etcd.yaml
```

Comment:
Hm, all these manifest links have a hardcoded `master` set.

How are we planning on supporting multiple release branches here? Can we just fill in the `master` bit with the correct `vX.Y` based on the branch?